packages/evergreen-component-mapper/src/combo-box/combo-box.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react';
+
+import { Combobox as EvergreenCombobox, FormField } from 'evergreen-ui';
+
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+import {
+	AnyObject,
+	UseFieldApiProps,
+} from '@data-driven-forms/react-form-renderer';
+
+export interface ComboBoxItem extends AnyObject {
+    value?: any;
+    label: string;
+}
+
+export interface ComboBoxProps extends UseFieldApiProps<string> {
+    name: string;
+    isRequired?: boolean;
+    options: ComboBoxItem[];
+}
+
+const ComboBox: React.FC<ComboBoxProps> = (props) => {
+	const {
+		id,
+		input,
+		meta,
+		isDisabled,
+		options,
+		isRequired,
+		label,
+		description,
+		inputProps,
+		...rest
+	} = useFieldApi(props);
+
+	const selectedItem = useMemo(() => {
+		return options.find((item: ComboBoxItem) => item.value === input.value);
+	}, [input.value, options]);
+
+	return (
+		<FormField
+			labelFor={id}
+			label={label}
+			description={description}
+			isRequired={isRequired}
+			validationMessage={meta.error}
+			marginBottom={24}
+		>
+			<EvergreenCombobox
+				id={id}
+				width="100%"
+				selectedItem={selectedItem}
```

Comment:
We got this error when selecting any value

<img width="721" alt="Snímek obrazovky 2023-08-24 v 9 12 18" src="https://github.com/data-driven-forms/editor/assets/32869456/06bc43a3-fc0c-4798-b681-ff1ff372d853">

we need to provide `null` as a default value instead of `undefined`